fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -1864,8 +1872,15 @@ public PlanFragment visitPhysicalProject(PhysicalProject<? extends Plan> project
         }
 
         PlanFragment inputFragment = project.child(0).accept(this, context);
-
         PlanNode inputPlanNode = inputFragment.getPlanRoot();
+        // this means already have project on this node, filter need execute after project, so need a new node
```

Comment:
Does it mean we may meet project - project plan ( why not be merged? ). So have to create a new select node to do the second project?

Reply:
yes, we should merge when do RBO, this is just in case